### PR TITLE
DISTMONGO-200 Convert docs to Markdown (5.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
Configure MkDocs for doc build
Configure Render for docs hosting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
 
 Percona Distribution for MongoDB consists of the following components:
 
-- [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/4.4/index.html) - a fully-compatible open-source replacement for MongoDB Community Edition.
+- [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/4.2/index.html) - a fully-compatible open-source replacement for MongoDB Community Edition.
```
Comment (nastena1606) [inline README.md]: The URL should include 5.0 here.Please kindly update it

File: README.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
...
 
-This repository contains the source files for the [Percona Distribution for MongoDB documentation](https://www.percona.com/doc/percona-distribution-for-mongodb/4.4/index.html).
+This repository contains the source files for the [Percona Distribution for MongoDB documentation](https://www.percona.com/doc/percona-distribution-for-mongodb/4.2/index.html).
 
 The documentation is written in [reStructured text markup language](https://docutils.sourceforge.io/rst.html) and is created using [Sphinx Python Documentation Generator](https://www.sphinx-doc.org/en/master/).
 
 ## Contributing
 
-We welcome all contributions and are always looking for new members that are as dedicated to serving the community as we are. You can reach out to us using our [forums](https://forums.percona.com/c/mongodb/percona-distribution-for-mongodb/43) and [Discord](https://discord.gg/mQEyGPkNbR).
+We welcome all contributions and are always looking for new members that are as dedicated to serving the community as we are. You can reach out to us using our [forums](https://forums.percona.com/c/postgresql/25) and [Discord](https://discord.gg/mQEyGPkNbR).
```
Comment (nastena1606) [inline README.md]: The URL leads to postgresql. It should lead to pdmdb

File: README.md [hunk 3]
```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
...
 
-For how to contribute to documentation, read the [Contributing guide](https://github.com/percona/distmongo-docs/blob/4.4/CONTRIBUTING.md).
+For how to contribute to documentation, read the [Contributing guide](https://github.com/percona/distmongo-docs/blob/4.2/CONTRIBUTING.md).
```
Comment (nastena1606) [inline README.md]: The version is 5.0

File: _resource/overrides/main.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+{# Import the theme's layout. #}
+{% extends "base.html" %}
+
+{%- macro relbar2 () %}
+<div class="md-relbar2__inner md-grid">
+<div class="md-content">
+  <article class="md-content__inner md-typeset" role="main">    
+    <h4>Contact Us </h4>
+       <p>For free technical help, visit the Percona <a class="reference external" href="https://forums.percona.com/c/postgresql/25?utm_campaign=Doc%20pages" target="_blank">Community Forum</a>.<br>
```
Comment (nastena1606) [inline _resource/overrides/main.html]: Change the URL to pdmdb one

File: _resource/overrides/main.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+ </article>
+</div>
+</div>
+{%- endmacro %}
+
+              {% block content %}
+
+                <!-- Edit and view button -->
+                {% if page.edit_url %}
+                  {% set edit = "https://github.com/percona/pbm-docs/edit/main/docs/" %}
```
Comment (nastena1606) [inline _resource/overrides/main.html]: The edit and icons source should be pdmdb instead of pbm

**Review by nastena1606 — COMMENTED** [review]
Let's get rid of PDMDB_links.png. It is redundant

File: README.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

File: _resource/overrides/main.html (hunk 4 above)
Comment (alina-derkach-oaza) [reply _resource/overrides/main.html]: Updated

File: _resource/overrides/main.html (hunk 5 above)
Comment (alina-derkach-oaza) [reply _resource/overrides/main.html]: Updated

File: README.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

**Pull request merged**